magento/pwa-studio
# [security] minimist >= 1.2.2

## Description

1) Migrated `pwa-devdocs` to `yarn` from `npm`.
2) Used `resolutions` to resolve security vulnerability in `pwa-devdocs` and `docker` packages.

## Related Issue





Closes [PWA-449](https://jira.corp.magento.com/browse/PWA-449).

## Acceptance 



### Verification Stakeholders

### Specification



### Verification Steps


1. Verify docker did not break.
2. Verify pwa-devdocs did not break.
3. Verify that vulnerable versions of minimist are no longer included.

## Screenshots / Screen Captures (if appropriate)

## Checklist

* I have updated the documentation accordingly, if necessary.
* I have added tests to cover my changes, if necessary.
